.vuepress/components/InstallInstructions.vue

```diff
@@ -155,12 +155,12 @@ usermod -a -G openhab myownuser
         &#128712; <small>	You don't need the add-ons archives if your machine has Internet access, openHAB will download add-ons you need online as necessary.</small>
         <div v-if="(selectedVersion === 'stable' || selectedVersion === 'testing')">
             <div class="download-button-container">
-              <a class="download-button" style="margin-bottom: 0" :href="addonsDownloadLink">Download openHAB {{currentDownloadVersion}} {{currentVersionLabel}} Add-ons</a>
+              <a class="download-button" style="margin-bottom: 0" :href="addonsDownloadLink" download>Download openHAB {{currentDownloadVersion}} {{currentVersionLabel}} Add-ons</a>
             </div>
         </div>
         <div v-if="selectedVersion === 'snapshot'">
             <div class="download-button-container">
-              <a class="download-button" style="margin-bottom: 0" :href="`https://ci.openhab.org/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-addons/target/openhab-addons-4.0.0-SNAPSHOT.kar`">Download openHAB {{$page.frontmatter.currentSnapshotVersion}} Add-ons</a>
+              <a class="download-button" style="margin-bottom: 0" :href="`https://ci.openhab.org/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-addons/target/openhab-addons-4.0.0-SNAPSHOT.kar`" download>Download openHAB {{$page.frontmatter.currentSnapshotVersion}} Add-ons</a>
```

Comment:
```suggestion
              <a class="download-button" style="margin-bottom: 0" :href="`https://ci.openhab.org/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab-addons/target/openhab-addons-4.0.0-SNAPSHOT.kar`" download="openhab-addons-{{$page.frontmatter.currentSnapshotVersion}}.kar">Download openHAB {{$page.frontmatter.currentSnapshotVersion}} Add-ons</a>
```

We could add the version dynamically too, so attribute would be used completely and the version still would be included.

Reply:
That works for the snapshot link, but the stable/milestone links are inserted dynamically - this would a bit more scripting to get it right...

Reply:
Then we should directly add additional methods for the different download file names right away.

Please leave the naming part out for now.
I will take care of it.